mypy/stubtest.py

```diff
@@ -1345,13 +1359,23 @@ def test_stubs(args: argparse.Namespace, use_builtins_fixtures: bool = False) ->
 
     modules = args.modules
     if args.check_typeshed:
-        assert not args.modules, "Cannot pass both --check-typeshed and a list of modules"
+        if args.modules:
+            print(
+                _style("error:", color="red", bold=True),
+                "cannot pass both --check-typeshed and a list of modules",
```

Comment:
Wouldn't a simple wrapper make this nicer?

Just something simple like:
```py
def _error(message: str) -> None:
    print(_style("error: ", color="red", bold=True), message)
```

Reply:
It's a question of taste. It's one line, it makes it very clear what it's doing and it's not repeated that much, so I think it's fine as is :-)